packages/firebase_dynamic_links/android/src/main/java/io/flutter/plugins/firebasedynamiclinks/FirebaseDynamicLinksPlugin.java

```diff
@@ -138,19 +183,22 @@ public void onSuccess(PendingDynamicLinkData pendingDynamicLinkData) {
               }
             })
         .addOnFailureListener(
-            registrar.activity(),
             new OnFailureListener() {
               @Override
-              public void onFailure(@NonNull Exception e) {
+              public void onFailure(Exception e) {
                 result.error(e.getClass().getSimpleName(), e.getMessage(), null);
               }
             });
   }
 
+  private Activity getActivity() {
+    return registrar != null ? registrar.activity() : activityBinding.getActivity();
+  }
+
   private OnCompleteListener<ShortDynamicLink> createShortLinkListener(final Result result) {
     return new OnCompleteListener<ShortDynamicLink>() {
       @Override
-      public void onComplete(@NonNull Task<ShortDynamicLink> task) {
```

Comment:
Were these causing issues?

Reply:
answered in https://github.com/FirebaseExtended/flutterfire/pull/1372#discussion_r345961234